packages/calypso-e2e/src/lib/flows/start-site-flow.ts

```diff
@@ -49,7 +49,11 @@ export class StartSiteFlow {
 	 * @param {string} name Name for the blog.
 	 */
 	async enterBlogName( name: string ): Promise< void > {
-		await this.page.fill( selectors.blogNameInput, name );
+		const locator = this.page.locator( selectors.blogNameInput );
+		// Use the type method to simulate user interaction to avoid
+		// https://github.com/Automattic/wp-calypso/issues/64271.
+		await locator.fill( '' );
+		await locator.type( name );
```

Comment:
From observing the trace files it looks like this may be case of the blog_name and blog_tagline inputs being entered prior to the page load being completely finished.

By replacing the `fill` method (which is nearly instant) with a `type` method (which takes some more time, plus it emits a typing event) I am hoping that this behavior can be avoided.